src/index.ts

```diff
@@ -87,3 +93,12 @@ export const getWorkData = ({
     collectionName: url.match(/collections\/(\w+)/)?.[1],
   };
 };
+
+export const getWork = async ({ workId }: { workId: string }): Promise<any> => {
+  const workPage = await getWorksPage(workId);
```

Comment:
We should rename this to `getWorkPage` (singular). The feed has multiple works, but the page only has one.